docs/tutorials/developer-journey/level-4/4.1-icp-ledger.mdx

```diff
@@ -276,7 +276,7 @@ To transfer tokens from one account to another, use the command:
 dfx ledger transfer --amount AMOUNT --memo MEMO RECEIVER_ACCOUNT_ID
 ```
 
-Replace `AMOUNT` with the number of tokens to transfer, `MEMO` with a brief message to describe the reason for the transfer, and `RECEIVER_ACCOUNT_ID` with the account ID to receive the tokens.
+Replace `AMOUNT` with the number of tokens to transfer, `MEMO` with a whole number value, and `RECEIVER_ACCOUNT_ID` with the account ID to receive the tokens.
```

Comment:
is there any intuition we can provide for why its called memo?

Reply:
Added additional context based on FI team input: https://dfinity.slack.com/archives/C01D7R95YJE/p1722922982683399?thread_ts=1722898458.801149&cid=C01D7R95YJE